tests/test_crs.py

```diff
@@ -314,12 +316,12 @@ def test_from_esri_wkt():
         'VDATUM["North_American_Vertical_Datum_1988"],'
         'PARAMETER["Vertical_Shift",0.0],'
         'PARAMETER["Direction",1.0],UNIT["Centimeter",0.01]]]')
-    proj_crs_str = CRS.from_string(projection_string)
-    proj_crs_wkt = CRS.from_wkt(projection_string)
-    assert proj_crs_str.to_string() == proj_crs_wkt.to_string()
-    assert proj_crs_str.to_string() == \
-        ("+datum=NAD83 +lat_0=23 +lat_1=29.5 +lat_2=45.5 "
-         "+lon_0=-96 +no_defs +proj=aea +units=m +x_0=0 +y_0=0")
+    with Env(GDAL_FIX_ESRI_WKT='DATUM'):
+        proj_crs_str = CRS.from_string(projection_string)
+        proj_crs_wkt = CRS.from_wkt(projection_string)
+        assert proj_crs_str.to_string() == proj_crs_wkt.to_string()
+        assert proj_crs_str['ellps'] == 'GRS80'
+        assert proj_crs_str['towgs84'] == '0,0,0,0,0,0,0'
```

Comment:
@snowman2 here's the impact of `GDAL_FIX_ESRI_WKT='DATUM'`. See https://www.gdal.org/classOGRSpatialReference.html#ad556dfdc04d9ec5f1714fc6b5e0eb6a6.

Reply:
It may be good to keep the original version of the test. Even if the MorphFromESRI function is not modifying parameters, it still enables the ESRI WKT to be loaded in properly and converted to a PROJ.4 string.